assignment_day_11/src/constants/styles.js

```diff
@@ -0,0 +1,17 @@
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
```

Comment:
Do not create separate files for styles. Look into the wrapper container component. Those are also used for the same purpose

Reply:
Ok sir
Will look into this